tests/testthat/test-shiny.R

```diff
@@ -0,0 +1,66 @@
+test_that("daisyTimelineOutput creates proper Shiny output", {
+  output <- daisyTimelineOutput("test-output")
+  
+  # The output from htmlwidgets::shinyWidgetOutput is typically a shiny.tag.list
+  expect_s3_class(output, "shiny.tag.list")
+  
+  # Check that it contains the expected HTML widget structure
+  expect_true(length(output) > 0)
+  
+  # The actual structure depends on htmlwidgets implementation
+  # Just verify it's a valid Shiny output
+  expect_true(is.list(output))
+})
+
+test_that("daisyTimelineOutput accepts custom dimensions", {
+  output <- daisyTimelineOutput("test-output", width = "80%", height = "300px")
+  
+  expect_s3_class(output, "shiny.tag.list")
+  expect_true(length(output) > 0)
+  
+  # The dimensions are passed to htmlwidgets::shinyWidgetOutput
+  # We can't easily test the exact CSS without rendering, but we can test
+  # that the function accepts the parameters without error
+  expect_true(is.list(output))
+})
```

Comment:
This feature is a little beyond the original scope.  I'm not sure this test is great.